app/src/tests/lib/auth/getAuthHeader.test.ts

```diff
@@ -0,0 +1,14 @@
+const mockGetBearerToken = jest.fn();
+jest.mock("../../../lib/auth/getBearerToken", () => ({
+    getBearerToken: () => mockGetBearerToken()
```

Comment:
```suggestion
    getBearerToken: jest.fn(() => "1234")
```
Can avoid creating another var

Reply:
Ah yeah, can do here actually since we don't need to explicitly expect the mock